deploy/kubernetes/configmap.yaml

```diff
@@ -368,7 +355,7 @@ data:
     # 'DENY' will enable logging only for DENY rules. Remove this config if no
     # logging is desired. When IPv6 is enabled this setting will not apply to
     # rules for allowing ND traffic.
-    # Choices: ALL DENY <None>
+    # Choices: ALL DENY None NONE none
```

Comment:
Same here

Reply:
These three values are from auto generated configmap yaml based recent ncp changes.So, it might be to keep as is